README.md

```diff
@@ -127,7 +127,23 @@ Description: a subroutine of `visibiity_polygon_demonstration.m` and
 `shortest_path_demo.m` used to read a VisiLibity style Environment file into a
 Matlab cell array.
 
+## Testing
+Testing frameworks are available for visibility polygon and shortest path
+calculations.
 
+### Visibility polygon calculation testing
+TBD
+
+### Shortest path calculation testing
+
+The script `run_sp_.sh` will execute all the tests in the `sp_tests` directory.
```

Comment:
There are many things "sp" could stand for, so I recommend using "shortest_path" explicitly rather than the abbreviation "sp", i.e., rename
"run_sp_.sh" --> "run_shortest_path_tests.sh" (or maybe just "test_shortest_path.sh"?)
"sp_tests" --> "shortest_path_tests" etc.

Is there a reason you have the trailing underscore on the shell script name?

Reply:
I actually agree. I was being too flippant. Names are changed!